src/helpers/daily-data-providers/combined-resting-heart-rate.ts

```diff
@@ -16,6 +21,9 @@ export default function (startDate: Date, endDate: Date) {
 		if (settings.queryableDeviceDataTypes.find(s => s.namespace == "AppleHealth" && s.type == "RestingHeartRate")) {
 			providers.push(appleHealthRestingHeartRateDataProvider(startDate, endDate));
 		}
+		if (settings.healthConnectEnabled) {
```

Comment:
We should also query MyDataHelps.getDeviceDataV2AllDataTypes() and check to make sure that resting heart rate is enabled for Health Connect before including it here